ebos/ecloutputblackoilmodule.hh

```diff
@@ -1115,13 +1116,13 @@ public:
     }
 
     // write production report to output 
-    void outputProdLog(size_t reportStepNum, const bool substep, bool forceDisableProdOutput) 
+    void outputProdLog(size_t reportStepNum, const bool substep) 
```

Comment:
There is a lot of 8-space indentation here, but fixing that would edit every line... So just leave it like that and we'll eventually do an automatic formatting sweep.